_sass/components/_blockquote-section.scss

```diff
@@ -0,0 +1,109 @@
+$bp-quote-sm: 62.5rem;
+$bp-quote-md: 86.25rem;
+$bp-quote-lg: 106.25rem;
+$bp-quote-xl: 125rem;
+
+.c-blockquote {
+  width: 100%;
+  padding: 6rem 0;
+  margin: 4rem auto;
+  background: #F4F4F4;
+
+  &__inner-container {
+    width: 80%;
+    margin: 0 auto;
+    position: relative;
+  }
+
+  &__image-container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+  }
+
+  &__image-container {
+    width: 100%;
+    max-width: 37.5rem;
+    margin: 0 auto 2rem;
+
+    @media (min-width: $bp-sm) {
+      margin: 0;
+    }
+
+    @media (min-width: $bp-quote-sm) {
+      max-width: 37.875rem;
+    }
+
+    @media (min-width: $bp-quote-md) {
+      max-width: 47.875rem;
+    }
+
+    @media (min-width: $bp-quote-lg) {
+      max-width: 59.875rem;
+    }
+
+    @media (min-width: $bp-quote-xl) {
+      max-width: 68.875rem;
+    }
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__gradient-container:after {
+    position: absolute;
+    width: 100%;
+    height: 99.4%;
+    background-image: linear-gradient(to right, rgba(255, 255, 255, 0) 40%, rgba(255, 255, 255, 1) 100%,);
+    left: 0;
+    top: 0;
+    content: '';
+  }
+
+  &__text-container {
+    margin: 2rem 0 0;
+  }
+
+  &__text {
+    margin: 0;
+    font-size: 1.375rem;
+    font-weight: 700;
+
+    @media (min-width: 50rem) {
+      position: absolute;
+      top: 5rem;
+      left: 26rem;
+      font-size: 1.5rem;
+    }
+
+    @media (min-width: $bp-quote-sm) {
+      font-size: 2rem;
+    }
+
+    @media (min-width: $bp-quote-md) {
+      top: 10rem;
+      left: 39rem;
+      font-size: 2rem;
```

Comment:
This `font-size` has already been declared on line 80, so you shouldn't need it here.

Reply:
oops